.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: java
 
+os:
+  - linux
+  - osx
+
 jdk:
+  - oraclejdk11
```

Comment:
Okay, so, I *think* this [might be causing issues on osx](https://docs.travis-ci.com/user/languages/java/#using-java-10-and-later)?

Reply:
I am not sure what you excactly mean.

> OracleJDK 11 and later are supported on Linux, and OpenJDK 10 and later are supported on Linux and macOS using install-jdk.sh.

Do you mean because Oracle JDK 11 is not mentioned for macOS that it is not supported?

It also worked once https://travis-ci.org/junit-pioneer/junit-pioneer/builds/652682481
It could be that travis Ci changed something and because of that it no longer works. So to test this it would be good to re run the previously working build again (I dont know how).
I suspect that because the working build had 2 macOS jobs but in the failing ones there is only 1 macOS job and I think the .travis.yml did not change.

Reply:
Seems to work. it should be removed anyway https://github.com/junit-pioneer/junit-pioneer/pull/174#discussion_r393946606

What I do not understand is that is once worked https://travis-ci.org/junit-pioneer/junit-pioneer/builds/652682481
https://travis-ci.org/github/junit-pioneer/junit-pioneer/jobs/652682485/config

Reply:
`travis_fold:start:install_jdk` was completely missing from the breaking build. It looked like Travis just skipped a step? Don't know why.